master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -1,142 +1,103 @@
 ---
-title: Going Beyond `NetworkPolicy` with Calico
+title: Going Beyond NetworkPolicy with Calico
+redirect_from: latest/getting-started/Kubernetes/tutorials/advanced-policy
```

Comment:
One thing I should have mentioned....you should omit the `redirect_from` in the master directory. Otherwise, we could get unexpected behavior with two redirects, but also, "master" is not considered the latest version. If you could just delete this line, that would be really great.